flink-core/src/test/java/org/apache/flink/api/java/typeutils/TypeExtractorTest.java

```diff
@@ -345,8 +346,25 @@ public CustomType cross(CustomType first, Integer second) throws Exception {
 
 		Assert.assertFalse(TypeExtractor.getForClass(PojoWithNonPublicDefaultCtor.class) instanceof PojoTypeInfo);
 	}
-	
 
+	@Test
+	public void testRow() {
```

Comment:
in different test method? 
there is already Row with a null field check

`Row nullRow = new Row(2);
TypeInformation<Row> genericRowInfo = TypeExtractor.getForObject(nullRow);
Assert.assertEquals(genericRowInfo, new GenericTypeInfo<>(Row.class));`